src/components/EventCard.js

```diff
@@ -14,18 +14,33 @@ import Text from "./Text";
 type Props = {
   name: string,
   locationName: string,
-  price: number,
+  eventPriceLow: number,
+  eventPriceHigh: number,
   startTime: string,
-  endTime: string
+  endTime: string,
+  isFree: boolean
 };
 const removeTimezoneFromDateString = isoString => isoString.slice(0, -6);
+const getEventPrice = (isFree, eventPriceLow, eventPriceHigh) => {
+  let displayPrice;
```

Comment:
You could get rid of the variable `displayPrice` and just return the strings from within the `if`.